src/mainwindow.cpp

```diff
@@ -182,7 +185,11 @@ MainWindow::MainWindow(QWidget* parent)
     connect(prettifySymbolsAction, &QAction::toggled, Settings::instance(), &Settings::setPrettifySymbols);
 
     ui->viewMenu->addSeparator();
-    ui->viewMenu->addActions(m_resultsPage->filterMenu()->actions());
+
+    QList<QAction *> actions = m_resultsPage->filterMenu()->actions();
+    // Disassembly menu items should not be presented here
+    actions.removeFirst();
```

Comment:
I thought it is rather natural to add it into context menu. Do you see better alternative?

Reply:
oh, certainly it should be part of the context menu, but as a separate action - not as part of the *filter menu*. this may require some other changes to make this new action fit in more naturally. but it has nothing to do with filtering. it's more like the other action to switch to the caller/callee view, have a look at how that is done please

Reply:
Ok.

Reply:
Don't you mind I keep Action disassembly in FilterAndZoomStack? It is used everywhere in setupContextMenu. For all pages.

Reply:
I don't see/understand why this would be necessary - just follow the approach taken by `jumpToCallerCallee` and be done with it no?

Reply:
To be able to call Disassembly from any tab - Bottom Up,  Top down, Caller / Callee through context menu of selected symbol. There is the method setupContextMenu of ResultUtil which is called for each tab. It has argument FilterAndZoomStack* filterStack. It seems to me better to add Disassembly action to the structure than propagate separately this action for every constructor for every class. I didn't catch your idea how to avoid this.